CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+
+## How to contribute code to colin
+
+### Reporting Bugs
+Before creating bug reports, please check a [list of known issues](https://github.com/user-cont/colin/issues) to see
+if the problem has already been reported.
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/user-cont/colin/issues/new).
+Be sure to include a **descriptive title, clear description and a package version** and please include **as many details as possible** to help maintainers reproduce
+the issue and resolve it faster.
+If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as GitHub issues.
+When you are creating an enhancement issue, **use a clear and descriptive title**
+and **provide a clear description of the suggested enhancement**
+in as many details as possible.
+
+# How to write the new check to colin
+All checks are stored in the directory [checks](https://github.com/user-cont/colin/tree/master/colin/checks).
+Think about what kind of check you would like to write and browse for already existing checks:
+ - [best practices](https://github.com/user-cont/colin/tree/master/colin/checks/best_practices)
+ - [containers](https://github.com/user-cont/colin/tree/master/colin/checks/containers)
+ - [dockerfile](https://github.com/user-cont/colin/tree/master/colin/checks/dockerfile)
+ - [images](https://github.com/user-cont/colin/tree/master/colin/checks/images)
+ - [labels](https://github.com/user-cont/colin/tree/master/colin/checks/labels)
+
+Here's a simple template how you can create a new check:
+
+## Label check example
+```python
+from colin.checks.abstract.labels import LabelCheck
+
+class FooBarLabelCheck(LabelCheck):
+
+    def __init__(self):
+        super(self.__class__, self) \
+            .__init__(name="foobar_label",
+                      message="Label 'foobar' has to be specified.",
+                      description="Provide a description for foobar label.",
+                      reference_url="Specify URL to foobar reference",
+                      tags=["foobar", "label", "required"],
+                      label="foobar",
+                      required=True,
+value_regex=None)
+```
+
+- `name` means what kind label we check. It has to end with `_label`.
+- `message` is shows in case check is not fulfilled.
+- `description` provides information, why the check has to be specified.
+- `reference_url` references to an URL for specific check.
+- `tags`
+- `label` means real label which is checked.
+- `required` specifies if label is required in case of True. Otherwise optional.
```

Comment:
@lachmanfrantisek Can you please look at it and correct sentences which are wrong?

Reply:
`label` -- name of the label, that will be checked
`required` -- no, in case of `True`, the label has to be there, otherwise, the label is not allowed (e.g. deprecated label). Do we have some better name for that?

You can also specify `value_regex`, that can be checked on the label value.